website_product_supplier/views/product_supplier_view.xml

```diff
@@ -0,0 +1,204 @@
+                                        <div class="text-center">
+                                            <label class="custom-file-input">
+                                                <div class="btn btn-default">Load Picture</div>
+                                                <input class="input-file" id="fileInput" type="file" name="ufile"/>
+                                            </label>
+                                        </div>
+                                    </div>
+                                    <div id="product_fields" class="col-md-8">
+                                        <section id="product_fields_base">
+                                            <div id="product_name" class="row">
+                                                <div t-attf-class="col-md-8 col-xs-12 form-group #{error.get('product_name') and 'has-error' or ''}">
+                                                    <label class="control-label">Name</label>
+                                                        <input id="product_name" class="form-control" name="product_name" placeholder="Product name..."
+                                                               t-att-value="supplierinfo.get('product_name')"/>
+                                                </div>
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('product_code') and 'has-error' or ''}">
+                                                    <label class="control-label">Code</label>
+                                                        <input id="product_code" class="form-control" name="product_code" placeholder="Product Code..."
+                                                               t-att-value="supplierinfo.get('product_code')"/>
+                                                </div>
+                                            </div>
+                                            <div id="supplier_info" class="row">
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('min_qty') and 'has-error' or ''}">
+                                                    <label class="control-label">Min Quantity</label>
+                                                    <div class="controls">
+                                                        <input id="min_qty" class="form-control" name="min_qty" placeholder="Min Quantity..."
+                                                               t-att-value="supplierinfo.get('min_qty')"/>
+                                                    </div>
+                                                </div>
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('delay') and 'has-error' or ''}">
+                                                    <label class="control-label">Delay</label>
```

Comment:
"Delay" is not very easy to understand by itself. Please use something more descriptive, or use some tooltips for those terms not easy to understand.